src/qmflows/parsers/cp2KParser.py

```diff
@@ -69,18 +73,22 @@ def assign_warning(warning_type: Type[Warning], msg: str,
 
 
 def read_cp2k_coefficients(path_mos: PathLike,
-                           plams_dir: Union[None, str, os.PathLike] = None) -> InfoMO:
+                           plams_dir: Optional_[PathLike] = None) -> InfoMO:
     """Read the MO's from the CP2K output.
 
     First it reads the number of ``Orbitals`` and ``Orbital`` functions from the
     cp2k output and then read the molecular orbitals.
 
     :returns: NamedTuple containing the Eigenvalues and the Coefficients
     """
-    file_out = fnmatch.filter(os.listdir(plams_dir), '*out')[0]
+    plams_dir = Path(plams_dir) if plams_dir else Path(os.getcwd())
+
     file_in = fnmatch.filter(os.listdir(plams_dir), '*in')[0]
-    path_in, path_out = [os.path.join(plams_dir, x)
-                         for x in [file_in, file_out]]
+    file_out = fnmatch.filter(os.listdir(plams_dir), '*out')[0]
+
```

Comment:
the '/' operator makes things much cleaner

Reply:
Yay, pathlib!